content/pt-br/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,90 @@
+ 
+O `status` descreve o _estado atual_ do objeto,  fornecido e atualizado pelo o sistema do Kubernetes e os componentes. O plano de controle do Kubernetes {{< glossary_tooltip text="control plane" term_id="control-plane" >}} gerência continuamente e ativamente o real estado para corresponder ao estado desejado que você forneceu.
+ 
+ 
+Por exemplo, no Kubernetes, o Deployment é um objeto que pode representar uma aplicação executando no seu cluster. Quando você cria o Deployment, você pode definir a `especificação` para especificar que você quer três réplicas da aplicação estejam rodando. O sistema de Kubernetes lê as especificações do Deployment e inicia três instâncias do seu aplicativo desejado, atualizando o status  para corresponder às suas especificações. Se uma dessas instâncias falhar (um status mudar), o sistema do Kubernetes responde as diferenças entre as especificações e o status fazendo uma correção--neste caso, iniciando uma instância de substituição.
+ 
+Para mais informações sobre especificações do objeto, status e metadados, veja Kubernetes [Kubernetes API Conventions](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md).
+ 
+
+
+
+
+### Descrevendo um objeto Kubernetes
+
+Quando se cria um objeto do Kubernetes, deve-se fornecer a especificação do objeto que descreve seu estado desejado, bem como algumas informações básicas sobre o objeto (como um nome, por exemplo). Quando utiliza a API Kubernetes para criar o objeto (diretamente ou via `kubectl`), essa solicitação de API deve incluir essa informação como JSON no corpo da solicitação. **Na maioria das vezes, você fornece as informações à `kubectl` em um arquivo .yaml**. `kubectl` converte a informação para JSON ao fazer a requisição para a API.
+
+Aqui está um exemplo de arquivo `.yaml` que mostra os campos necessários e as especificações de objeto para uma implatação Kubernetes:
+
+
+{{< codenew file="application/deployment.yaml" >}}
+
+Uma maneira de criar um Deployment usando um arquivo `.yaml` como o representado acima é usar o comando [`aplicar kubectl`](/docs/reference/generated/kubectl/kubectl-commands#apply
+) na interface de linha de comando `kubectl`, passando o arquivo `.yaml` como argumento. Aqui está um exemplo:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/deployment.yaml
+```
+
+A saída será similar a esta:
+
+```
+deployment.apps/nginx-deployment created
+```
+
+### Campos obrigatórios
+
+No arquivo `.yaml` para o objeto Kubernetes que pretende criar, você precisará definir valores para os seguintes campos:
+
+`apiVersion` - Qual a versão do API Kubernetes que está usando para criar este objeto.
```

Comment:
This statement is not correct. The correct would be "Qual a versão de API do objeto que será usado no Kubernetes para criar esse objeto"